app/services/sign_in/session_refresher.rb

```diff
@@ -59,6 +60,16 @@ def parent_refresh_token_in_session?
       session.hashed_refresh_token == double_parent_refresh_token_hash
     end
 
+    def validate_mpi_profile
+      mpi_service = MPI::Service.new
+      mpi_profile = mpi_service.find_profile_by_identifier(identifier: session.user_account&.icn,
```

Comment:
Does Rubocop allow for a new line after `(` and then putting `)&.profile` on its own line so you don't have to super indent the kwargs?

Reply:
It does! Much cleaner this way.